src/networking/payloads/connectiontype.rs

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
I would suggest you guys spend some times to configure some pre-commit git hooks to automatically format your code. This is well worth the effort, since you're going to do a lot of changes, write a lot of new files, and you really don't want to spend your precious mental energy on formatting issues...